### PR TITLE
charge interactable

### PR DESCRIPTION
**Pull request by LuigiPenco93**

### CODE REVIEW

File: ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/affordances/quickATs/RDXQuickATManager.java [hunk 1]
```diff
@@ -115,40 +115,43 @@ public void renderImGuiWidgets()
          quickATDirectory.renderImGuiWidgetsAsDropDownMenu();
       }
 
-      if (!selectedNode.equals(defaultNode))
+      if (selectedNode != null)
```
Comment (LuigiPenco93) [inline ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/affordances/quickATs/RDXQuickATManager.java:118]: this is an unrelated fix for a crash of the UI in the quick AT menu

File: ihmc-high-level-behaviors/src/libgdx/resources/environmentObjects/charge/color.jpg
Comment (calvertdw) [inline ihmc-high-level-behaviors/src/libgdx/resources/environmentObjects/charge/color.jpg:1]: I would reduce the size of this to kBs. 1.5 mB is way overkill

File: ihmc-perception/src/main/java/us/ihmc/perception/sceneGraph/rigidBody/RigidBodySceneObjectDefinitions.java [hunk 2]
```diff
@@ -297,10 +305,11 @@ public static void ensureBikeNodeAdded(SceneGraph sceneGraph, SceneGraphModifica
                                                         BIKE_NAME,
                                                         sceneGraph.getIDToNodeMap(),
                                                         parentNode.getID(),
-                                                        new RigidBodyTransform(), BIKE_VISUAL_MODEL_FILE_PATH,
+                                                        new RigidBodyTransform(),
+                                                        BIKE_VISUAL_MODEL_FILE_PATH,
                                                         BIKE_VISUAL_MODEL_TO_NODE_FRAME_TRANSFORM,
                                                         sceneGraph.getCRDTInfo());
-      LogTools.info("Adding Bike to scene graph.");
+      LogTools.info("Adding {} to scene graph.", BIKE_NAME);
```
Comment (calvertdw) [inline ihmc-perception/src/main/java/us/ihmc/perception/sceneGraph/rigidBody/RigidBodySceneObjectDefinitions.java:312]: There's a bike? lol

**Review by calvertdw — APPROVED** [review]

**Review by ds58 — APPROVED** [review]
